lib/PasswordHash/PasswordHash.php

```diff
@@ -30,7 +30,7 @@ class PasswordHash {
 	var $portable_hashes;
 	var $random_state;
 
-	function PasswordHash($iteration_count_log2, $portable_hashes)
+	function Password($iteration_count_log2, $portable_hashes)
```

Comment:
I'm pretty sure this should be changed to:

```
function __construct($iteration_count_log2, $portable_hashes)
```

See here for reference: https://stackoverflow.com/questions/37100373/php-deprecated-methods-with-the-same-name#37100413